glommio/benches/nop.rs

```diff
@@ -0,0 +1,99 @@
+//! Benchmark the noop performance of Glommio. This is a stress test for the executor/reactor
```

Comment:
this isn't measuring the performance "of glommio", but rather just a part of it (the submission to the ring)
Just a nitpick